handler/src/main/java/io/netty/handler/logging/LoggingHandler.java

```diff
@@ -304,6 +304,14 @@ public void flush(ChannelHandlerContext ctx) throws Exception {
         ctx.flush();
     }
 
+    @Override
+    public void read(ChannelHandlerContext ctx) throws Exception {
```

Comment:
Apparently we need to add a revapi override for the `@Skip` annotation being removed:

```
Error:  Failed to execute goal org.revapi:revapi-maven-plugin:0.15.0:check (api-check) on project netty-handler: The following API problems caused the build to fail:
Error:  java.annotation.removed: method void io.netty.handler.logging.LoggingHandler::read(io.netty.channel.ChannelHandlerContext) throws java.lang.Exception: Element no longer annotated with 'io.netty.channel.ChannelHandlerMask.Skip'. https://revapi.org/revapi-java/differences.html#java.annotation.removed
```